src/App.tsx

```diff
@@ -1,24 +1,55 @@
 import React from 'react';
 import './App.scss';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+import { Clock } from './Components/Clock/Clock';
+
+type State = {
+  isVisibleClock: boolean
 };
 
+class App extends React.Component<{}, State> {
+  state: State = {
+    isVisibleClock: true,
+  };
+
+  clockVisibility(isVisible: boolean): void {
+    this.setState({ isVisibleClock: isVisible });
+  }
+
+  render() {
+    const { isVisibleClock } = this.state;
+
+    return (
+      <div className="App">
+        <h2>React clock</h2>
+        {isVisibleClock && (
+          <div className="App__clock clock">
+            <h3>Current time:</h3>
+            {' '}
+            <div className="clock__time">
+              <Clock />
+            </div>
+          </div>
+        )}
+
+        <button
+          className="App__button"
+          type="button"
+          onClick={() => this.clockVisibility(false)}
```

Comment:
I think it will be better to create two different functions inside the class body. try to avoid anonymous functions